akka-docs/src/main/paradox/typed/routers.md

```diff
@@ -62,6 +62,17 @@ Scala
 Java
 :  @@snip [RouterTest.java](/akka-actor-typed-tests/src/test/java/jdocs/akka/typed/RouterTest.java) { #pool-dispatcher }
 
+### Broadcasting a message to all routees
+
+Pool routers can be configured to identify messages intended to be broad-casted to all routees.
+Therefore, the `PoolRouter` has a property to configure its `broadcastPredicate`:
+
+Scala
+:  @@snip [RouterSpec.scala](/akka-actor-typed-tests/src/test/scala/docs/akka/typed/RouterSpec.scala) { #broadcast }
+
+Java
+:  @@snip [RouterTest.java](/akka-actor-typed-tests/src/test/java/jdocs/akka/typed/RouterTest.java) { #broadcast }
+
 ## Group Router
```

Comment:
Shall group routers have the same feature?

Reply:
@patriknw if you think this has value for group routers as well I can add it to this pr or immediately start a new one.
I was focused on Pool routers since this was my use-case (implemented this using interception for the my requirements), group routers kind of support this ootb since they are based on the receptionist which means user code has access to all routees. btw seems like there's another complication for group routers regarding `preferLocalRoutees`, should broadcast reach remotes as well? 
another diff is that unlike pool routers the set of routees is not fixed, once a pool router broadcasted a command you know it reached all active routees - this does not hold for group routers where routees can later join, so assuming the broadcasted message is meaningful user-code will have to listen on the receptionist `Listing` messages in order make sure the entire group is 'on the same page'.

Reply:
That can be a separate PR.

> group routers kind of support this ootb since they are based on the receptionist which means user code has access to all routees

yes, but that would be a different API, and somewhat inconvenient if a group router is already used for some messages

> preferLocalRoutees, should broadcast reach remotes as well?

I think it should ignore that for broadcast, i.e. send to all remotes as well

> assuming the broadcasted message is meaningful user-code will have to listen on the receptionist Listing messages in order make sure the entire group is 'on the same page'

The GroupRouterImpl is already subscribing so it can keep track of latest list of routees

Wdyt about broadcast for group routers, @johanandren ?

Reply:
I thought it wasn't necessary twice before, but sure, it would make it a bit easier than a receptionist lookup + tell each, and it's probably easy enough to implement.